src/plugins/cosmos/components/modals/GetStarted/views/LearnMore.tsx

```diff
@@ -0,0 +1,151 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, IconButton } from '@chakra-ui/react'
+import { useSteps } from 'chakra-ui-steps'
+import { DefiModalHeader } from 'plugins/cosmos/components/DefiModalHeader/DefiModalHeader'
+import { useHistory } from 'react-router-dom'
+import rewards from 'assets/rewards.svg'
+import risk from 'assets/risk.svg'
+import withdraw from 'assets/withdraw.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+
+const STEPS_LENGTH = 3
+const STEP_TO_ELEMENTS_MAPPING = {
+  1: {
+    bodies: [
+      'defi.modals.learnMore.bodies.rateFluctuationInfo',
+      'defi.modals.learnMore.bodies.amountStakingInfo',
+      'defi.modals.learnMore.bodies.withdrawInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.aboutStakingRewards',
+    headerImageSrc: rewards
+  },
+  2: {
+    bodies: ['defi.modals.learnMore.bodies.unbondingInfo'],
+    header: 'defi.modals.learnMore.headers.unstaking',
+    headerImageSrc: withdraw
+  },
+  3: {
+    bodies: [
+      'defi.modals.learnMore.bodies.slashingInfo',
+      'defi.modals.learnMore.bodies.partnerInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.risks',
+    headerImageSrc: risk
+  }
+}
+
+type LearnMoreProps = {
+  assetId: string
+}
+
+export const LearnMore = ({ assetId }: LearnMoreProps) => {
+  const history = useHistory()
+  const { cosmosGetStarted } = useModal()
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo'
+  }))(assetId)
+
+  const { nextStep, prevStep, activeStep } = useSteps({
+    initialStep: 1
+  })
+
+  // activeStep from useSteps() can have an infinite of step, but at runtime, it is effectively restricted to max. 3 in our case
+  // This makes tsc happy regarding exhaustive property access of STEP_TO_ELEMENTS_MAPPING
+  let currentStep = activeStep as 1 | 2 | 3
+
+  const isLastStep = currentStep === STEPS_LENGTH
+  const isFirstStep = currentStep === 1
+
+  const handleNextOrCloseClick = () => {
+    if (isLastStep) return cosmosGetStarted.close()
+
+    nextStep()
+  }
+
+  const handlePrevClick = () => {
+    if (isFirstStep) {
+      return history.goBack()
+    }
+
+    prevStep()
+  }
+
```

Comment:
We can put a helper `const` here to make the JSX more concise:

```ts
const currentElement = STEP_TO_ELEMENTS_MAPPING[activeStep - 1]
```

Note the offset to `activeStep` so that we start at the beginning of our array.

Reply:
Done